static/js/subs.js

```diff
@@ -478,6 +478,25 @@ export function redraw_left_panel(left_panel_params = get_left_panel_params()) {
         widgets.set(stream_id, $(row).detach());
     }
 
+    let fl = true;
+
+    // hidden_ids size is always 14 when there is no stream match with the searched stream name.
```

Comment:
The `hidden_ids` is not always 14, but depends on the number of streams present in the realm. A realm having `x` streams will have the `hidden_ids.size` as `x` (and not 14).

Also keeping a variable name as `f1` is not intuitive. It should be kept signifying the data it holds/role it performs.